src/applications/vaos/new-appointment/components/ReviewPage/ReviewDirectScheduleInfo.jsx

```diff
@@ -30,7 +31,14 @@ export default function ReviewDirectScheduleInfo({
       {facility.address?.city}, <State state={facility.address?.state} />
       <ReasonForAppointmentSection data={data} />
       <hr aria-hidden="true" className="vads-u-margin-y--2" />
-      <ContactDetailSection data={data} flowType={FLOW_TYPES.DIRECT} />
```

Comment:
Removed 'flowType' prop to match other calls to the component that don't pass 'flowType'. The information can be retrieved via useSelector hook.